JustSaying.IntegrationTests/WhenRegisteringHandlersViaResolver/WhenRegisteringAHandlerWithMessageContext.cs

```diff
@@ -0,0 +1,47 @@
+using System.Threading;
+using System.Threading.Tasks;
+using JustSaying.IntegrationTests.TestHandlers;
+using JustSaying.TestingFramework;
+using Shouldly;
+using StructureMap;
+using Xunit;
+
+namespace JustSaying.IntegrationTests.WhenRegisteringHandlersViaResolver
+{
+    [Collection(GlobalSetup.CollectionName)]
```

Comment:
It would be nice if new tests used the new builder pattern so I've got less to migrate from the old world to the new...